src/app/SharpRaven/Data/Requester.Net45.cs

```diff
@@ -30,13 +30,15 @@
 
 using System.IO;
 using System.Net;
+#if (!net35)
 using System.Threading.Tasks;
+#endif
 
 using Newtonsoft.Json;
 
 using SharpRaven.Utilities;
 
-#if !(net40)
+#if !(net40) && !(net35)
```

Comment:
I am not sure. Currently that means it won't be taken into account for
those two platforms. In the future, more .net versions will appear and the
current code should work. If we force the use of #*if (net45)* we could be
forced to include explicitly future labels for future platforms?

On Wed, Sep 14, 2016 at 1:17 PM, Asbjørn Ulsberg <notifications@github.com>
wrote:

> In src/app/SharpRaven/Data/Requester.Net45.cs
> <https://github.com/getsentry/raven-csharp/pull/156#discussion_r78793033>:
>
> >
> >  using Newtonsoft.Json;
> >
> >  using SharpRaven.Utilities;
> >
> > -#if !(net40)
> > +#if !(net40) && !(net35)
>
> Perhaps a #if (net45) check would be neater?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/getsentry/raven-csharp/pull/156/files/8c07fedf94987f3aed3ae9a98d168c4540972b78#r78793033>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKtV5s9DYxXwkvph-PDZydfEyiZoa-yrks5qqCwngaJpZM4J8_tK>
> .
>